src/vivarium/framework/values.py

```diff
@@ -111,10 +111,15 @@ def rescale_post_processor(value: NumberLike, manager: "ValuesManager") -> Numbe
 
     """
     if hasattr(value, "index"):
-        time_step = manager.simulant_step_sizes(value.index).dt
+        return value.mul(
+            manager.simulant_step_sizes(value.index)
+            .astype("timedelta64[ns]")
+            .dt.total_seconds()
```

Comment:
isn't there a more direct `dt` method you can use instead of seconds and then have to convert?

Reply:
I copied this from `from_yearly` but yeah i think you could just do `step_size / timedelta(days=365)` and that would work the same.